multilinear-util/src/lib.rs

```diff
@@ -1,5 +1,6 @@
-#![no_std]
+// #![no_std]
```

Comment:
We want to keep this #![no_std]
```suggestion
#![no_std]
```